docs/api/web-contents.md

```diff
@@ -1416,6 +1420,46 @@ Executes the editing command `selectAll` in web page.
 
 Executes the editing command `unselect` in web page.
 
+#### `contents.scrollToTop()`
+
+Scrolls to the top of the current `webContents`.
+
+#### `contents.scrollToBottom()`
+
+Scrolls to the bottom of the current `webContents`.
+
+#### `contents.adjustSelection(options)`
+
+* `options` Object
```

Comment:
question: are `start` and `end` both mandatory properties for a reason here?

Reply:
I suppose we could just default to 0, yeah!